docs/reference/migration/migrate_8_0/reindex.asciidoc

```diff
@@ -0,0 +1,9 @@
+[float]
+[[breaking_80_reindex_changes]]
+=== Reindex changes
+
+Reindex from remote would previously allow URL encoded index-names and not re-encode them when generating the search
```

Comment:
This is a drive-by comment only, but we like to wrap our docs at eighty columns. 😄

Reply:
Thanks @jasontedor , fixed in 67dd80f6f487e259454f24a8efc653c60df7ca10.